docs/content.zh/docs/dev/table/sql/create.md

```diff
@@ -149,15 +149,15 @@ Flink SQL> INSERT INTO RubberOrders SELECT product, amount FROM Orders WHERE pro
 ##  CREATE TABLE
 
 ```text
-CREATE TABLE [IF NOT EXISTS] [catalog_name.][db_name.]table_name
+CREATE [TEMPORARY] TABLE [IF NOT EXISTS] [catalog_name.][db_name.]table_name
```

Comment:
I guess we should include MATERIALIZED as well like this [TEMPORARY | MATERIALIZED ]

Reply:
CREATE MATERIALIZED TABLE statement cannot have column definitions, and MATERIALIZED TABLE has a separate page for description, so I think this page is to create a normal table

Reply:
@jiefei30 that makes sense - though it is strange to have 2 create table grammars described. I suggest some text to link to the materialize table grammar would be useful. WDYT?

Reply:
@davidradl I agree. I will add some text to link to the materialized table grammar and temporary table description